src/Raven.Server/Utils/BackupUtils.cs

```diff
@@ -138,7 +138,7 @@ private static List<string> AddDestinations(PeriodicBackupStatus backupStatus)
     public static NextBackup GetNextBackupDetails(NextBackupDetailsParameters parameters)
     {
         var nowUtc = SystemTime.UtcNow;
-        var lastFullBackupUtc = parameters.BackupStatus.LastFullBackupInternal ?? parameters.DatabaseWakeUpTimeUtc ?? nowUtc;
+        var lastFullBackupUtc = parameters.BackupStatus.LastFullBackupInternal ?? parameters.DatabaseWakeUpTimeUtc ?? parameters.Configuration.CreatedAt;
```

Comment:
`CreatedAt` will be null for existing backup tasks

Reply:
I did not make `CreatedAt` nullable, so `DateTime.MinValue` will be its default value...

However, it might make sense to render this property nullable so that for existing backup tasks, we would have null there instead of `DateTime.MinValue`. And if `CreatedAt` is null, we would revert to using the previously applied current `nowUtc` time.

In this case, we would be fine with null, as `CreatedAt` will only be used for calculating all newly created backup tasks, while the last complete (or incremental) backup time will be utilized for calculating the next backup time for existing tasks.

Reply:
Done.